{"url":"https://api.github.com/repos/owncloud/core/issues/915","labels_url":"https://api.github.com/repos/owncloud/core/issues/915/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/915/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/915/events","html_url":"https://github.com/owncloud/core/issues/915","id":9311343,"number":915,"title":"oc-1713  -  [remotestorage] fails to support external user auth","user":{"login":"msrex","id":2625328,"avatar_url":"https://gravatar.com/avatar/24738c0908dad69b83af9cc534bcca80?d=https%3a%2f%2fidenticons.github.com%2ff02814cb1648a9552774c0f932619e91.png&r=x","gravatar_id":"24738c0908dad69b83af9cc534bcca80","url":"https://api.github.com/users/msrex","html_url":"https://github.com/msrex","followers_url":"https://api.github.com/users/msrex/followers","following_url":"https://api.github.com/users/msrex/following{/other_user}","gists_url":"https://api.github.com/users/msrex/gists{/gist_id}","starred_url":"https://api.github.com/users/msrex/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/msrex/subscriptions","organizations_url":"https://api.github.com/users/msrex/orgs","repos_url":"https://api.github.com/users/msrex/repos","events_url":"https://api.github.com/users/msrex/events{/privacy}","received_events_url":"https://api.github.com/users/msrex/received_events","type":"user","site_admin":false},"labels":[{"url":"https://api.github.com/repos/owncloud/core/labels/bug","name":"bug","color":"fc2929"}],"state":"closed","assignee":null,"milestone":null,"comments":1,"created_at":"2012-12-15t21:58:03z","updated_at":"2013-02-28t19:09:08z","closed_at":"2013-02-28t19:09:08z","pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"from http://bugs.owncloud.org/thebuggenie/owncloud/issues/oc-1713\r\n\r\nunfortunately i can't judge whether this is still relevant. there is a patch in the original bugreport.\r\n\r\ndescription:\r\ni'm using the user_ldap and failed to get remotestorage links (empty array) for existing users. this is due to other apps not being loaded, thus oc_user::userexists is only propagated to the oc_user_database class and never to other backends because they could not register into _usedbackends. the attached patch fixes this problem.\r\n\r\n\r\n\r\nreproduction steps:\r\nn/a.\r\n\r\n#2 comment posted by blizzz sep 26, 16:31\r\nwhich version? why the change in webfinger?\r\n\r\n#3 comment posted by duck sep 26, 18:07\r\nversion 4.0.7\r\n\r\nyou're probably right, changing the logic in public.php instead should solve this kind of problems globally. i guess calling oc_app::loadapps() instead of oc_app::loadapp($app) should not have any drawback. \r\n","closed_by":{"login":"jancborchardt","id":925062,"avatar_url":"https://gravatar.com/avatar/2fd3f4d5d762955e5b603794a888fa97?d=https%3a%2f%2fidenticons.github.com%2f84ec51f9ceed468217d4e13abf78da9c.png&r=x","gravatar_id":"2fd3f4d5d762955e5b603794a888fa97","url":"https://api.github.com/users/jancborchardt","html_url":"https://github.com/jancborchardt","followers_url":"https://api.github.com/users/jancborchardt/followers","following_url":"https://api.github.com/users/jancborchardt/following{/other_user}","gists_url":"https://api.github.com/users/jancborchardt/gists{/gist_id}","starred_url":"https://api.github.com/users/jancborchardt/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/jancborchardt/subscriptions","organizations_url":"https://api.github.com/users/jancborchardt/orgs","repos_url":"https://api.github.com/users/jancborchardt/repos","events_url":"https://api.github.com/users/jancborchardt/events{/privacy}","received_events_url":"https://api.github.com/users/jancborchardt/received_events","type":"user","site_admin":false}}